crates/swc_common/src/plugin.rs

```diff
@@ -8,8 +8,9 @@ use crate::{syntax_pos::Mark, SyntaxContext};
 use anyhow::Error;
 use std::any::type_name;
 
+/// Serialize ast to pass into plugins. This is being called from host side.
 /// TODO: https://github.com/swc-project/swc/issues/3167
-#[cfg(feature = "plugin-mode")]
+#[cfg(feature = "plugin-rt")]
 pub fn serialize_for_plugin<T>(t: &T) -> Result<rkyv::AlignedVec, Error>
```

Comment:
It think it should be `plugin-base`, as both of runtime and plugin require this.

Reply:
Previously it was used by both. (Before removing codes in `swc_common/plugin`.

Reply:
yes, you're correct. I only realized right after changing this but went sleep. 😅